### PR TITLE
MAIN - adds img header on missing pages

### PR DESCRIPTION
**Pull request by paschalidi**

### CODE REVIEW

File: src/Welcome/EthosPage.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { Row, Col } from "../../src/index";
 import WelcomeStyles from "./welcome.style";
+import header from "../../public/images/header.png";
```
Comment (herman-rogers) [inline src/Welcome/EthosPage.jsx]: is this being used?

Comment (james-ciclitira) [reply src/Welcome/EthosPage.jsx]: I removed to keep the page looking the same

**Comment by paschalidi** [comment]
@herman-rogers when I take your agreement I will go ahead and close this PR

**Comment by herman-rogers** [comment]
Cool let's just close this.

**Pull request closed without merging**